src/pages/home/report/ReportActionItem.js

```diff
@@ -278,8 +278,15 @@ class ReportActionItem extends Component {
 
         const reactions = _.get(this.props, ['action', 'message', 0, 'reactions'], []);
         const hasReactions = reactions.length > 0;
+
+        const numberOfThreadReplies = _.get(this.props, ['action', 'childVisibleActionCount'], 0);
```

Comment:
NAB: I think we prefer to avoid using the get function if we can use short circuiting instead, so you could change it to this, like what we passed to ReportActionItemThread before.
```javascript
this.props.action.childVisibleActionCount || 0;
```

Reply:
I think I'm going to leave this one as is